packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -2,9 +2,1438 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 import 'package:meta/meta.dart';
 
-import 'print.dart';
+/// Styles for displaying a node in a [DiagnosticsNode] tree.
+///
+/// See also:
+///
+///  * [DiagnosticsNode.toStringDeep], which dumps text art trees for these
+///    styles.
+///  * [_normalTextConfiguration], which shows what text art for the standard
```

Comment:
Eliminated all taunting from tree_diagnostics_mixin.dart
make boxTextConfiguration, etc public as there isn't a good reason not to expose them for users who want to create their own custom DiagnosticsNode formats.
I'm open to moving them into a separate library that does not get exported by default as that would also be reasonable.